src/components/application_manager/include/application_manager/application_manager_impl.h

```diff
@@ -1705,6 +1721,9 @@ class ApplicationManagerImpl
   ServiceStreamingStatusMap streaming_application_services_;
   sync_primitives::Lock streaming_services_lock_;
 
+  ExpiredButtonRequestsMap expired_button_requests_;
+  mutable sync_primitives::Lock expired_button_requests_lock_;
```

Comment:
@ychernysheva please declare lock before map which it protects

Reply:
Done in 950d7d4